### ISSUE TITLE
Windows 10 x64: Error during building of robotframework

### ISSUE THREAD

**Issue by bmcguirk** [conda/conda-build#1060]
Please see below: 

```
PS pkg:\> conda skeleton pypi robotframework
Warning, the following versions were found for robotframework
2.0
2.0.1
2.0.2
2.0.3
2.0.4
2.1
2.1.1
2.1.2
2.1.3
2.5
2.5.1
2.5.2
2.5.3
2.5.5
2.5.6
2.5.7
2.6.1
2.6.2
2.6.3
2.7
2.7.1
2.7.2
2.7.3
2.7.4
2.7.5
2.7.6
2.7.7
2.8
2.8.1
2.8.2
2.8.3
2.8.4
2.8.5
2.8.6b1
2.8.6
2.8.7
2.9a1
2.9a2
2.9a3
2.9b1
2.9b2
2.9rc1
2.9
2.9.1
2.9.2
3.0a1
3.0a2
3.0b1
3.0
Using 2.0
Use --version to specify a different version.
Error: Could not build recipe for robotframework. Could not find any valid urls.
PS pkg:\> conda skeleton pypi robotframework --version 3.0
Using url https://files.pythonhosted.org/packages/93/15/35b58d06f35e2c12fc170c8d2aa46a8617beca52d240849388d9bf0f8386/robotframework-3.0.tar.gz (420 KB) for robotframework.
Downloading robotframework
Unpacking robotframework...
done
working in C:\Users\BRIAN~1.MCG\AppData\Local\Temp\tmpvx2eccvfconda_skeleton_robotframework-3.0.tar.gz
Fetching package metadata .........
Solving package specifications: ..........

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    vs2015_runtime-14.0.25123  |                0         1.9 MB
    python-3.5.1               |                5        29.6 MB
    numpy-1.11.0               |           py35_2         2.8 MB
    ------------------------------------------------------------
                                           Total:        34.3 MB

The following NEW packages will be INSTALLED:

    mkl:            11.3.3-1
    numpy:          1.11.0-py35_2
    pip:            8.1.2-py35_0
    python:         3.5.1-5
    pyyaml:         3.11-py35_4
    setuptools:     23.0.0-py35_0
    vs2015_runtime: 14.0.25123-0
    wheel:          0.29.0-py35_0
    yaml:           0.1.6-0

Fetching packages ...
vs2015_runtime 100% |###############################| Time: 0:00:00   5.48 MB/s
python-3.5.1-5 100% |###############################| Time: 0:00:04   6.37 MB/s
numpy-1.11.0-p 100% |###############################| Time: 0:00:00   3.62 MB/s
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
[      COMPLETE      ]|##################################################| 100%
Applying patch: 'C:\\Users\\BRIAN~1.MCG\\AppData\\Local\\Temp\\tmpvx2eccvfconda_skeleton_robotframework-3.0.tar.gz\\pypi-distutils.patch'
patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 1 line).
Writing recipe for robotframework
Done
PS pkg:\> conda build robotframework
Removing old build environment
BUILD START: robotframework-3.0-py35_0
Fetching package metadata .........
Solving package specifications: ..........

The following NEW packages will be INSTALLED:

    pip:            8.1.2-py35_0
    python:         3.5.1-5
    setuptools:     23.0.0-py35_0
    vs2015_runtime: 14.0.25123-0
    wheel:          0.29.0-py35_0

Linking packages ...
[      COMPLETE      ]|##################################################| 100%
Source cache directory is: C:\Anaconda3\conda-bld\src_cache
Found source in cache: robotframework-3.0.tar.gz
Extracting download
Package: robotframework-3.0-py35_0
source tree in: C:\Anaconda3\conda-bld\work\robotframework-3.0
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda-build/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Anaconda3\Scripts\conda-build-script.py", line 4, in <module>
    sys.exit(main())
  File "C:\Anaconda3\lib\site-packages\conda_build\main_build.py", line 144, in main
    args_func(args, p)
  File "C:\Anaconda3\lib\site-packages\conda_build\main_build.py", line 389, in args_func
    args.func(args, p)
  File "C:\Anaconda3\lib\site-packages\conda_build\main_build.py", line 332, in execute
    dirty=args.dirty, activate=args.activate)
  File "C:\Anaconda3\lib\site-packages\conda_build\build.py", line 563, in build
    import conda_build.windows as windows
  File "C:\Anaconda3\lib\site-packages\conda_build\windows.py", line 30, in <module>
    'installationfolder')
  File "C:\Anaconda3\lib\distutils\msvc9compiler.py", line 70, in get_value
    raise KeyError(key)
KeyError: 'installationfolder'
PS pkg:\> conda info
Current conda install:

             platform : win-64
        conda version : 4.1.4
    conda-env version : 2.5.0
  conda-build version : 1.21.2
       python version : 3.5.1.final.0
     requests version : 2.9.1
     root environment : C:\Anaconda3  (writable)
  default environment : C:\Anaconda3
     envs directories : C:\Anaconda3\envs
        package cache : C:\Anaconda3\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : C:\Users\Brian.McGuirk\.condarc
         offline mode : False
    is foreign system : False

PS pkg:\>

```

Pay no attention to #974 . Any help would be appreciated.

**Comment by msarahan** [comment]
Fixed in Conda-build 1.21.3 (see #1058 #1048).  Has been tagged, but packages are not yet available.  Should be soon.

**Comment by bmcguirk** [comment]
@msarahan Nice! That was fast. Thanks.
